server/src/controllers/seasonTicketTypePriceDefaultController.ts

```diff
@@ -299,7 +312,29 @@ seasonTicketTypePriceDefaultController.put('/:seasonid', async (req: Request, re
         })
         .flat(1),
     ));
-    return res.json(await prisma.seasontickettypepricedefault.findMany({where: {seasonid_fk: +seasonid}}));
+    const toSend = await prisma.seasontickettypepricedefault.findMany({
+      where: {
+        seasonid_fk: +seasonid,
```

Comment:
What is the `+` mean? Is this a typo?

Reply:
The `+` unary operator converts whatever it precedes to a number

Reply:
@henrykaus Why would we use that over Number() to convert to a number? It definitely looks more concise but just curious if there are any side effects with it.

Reply:
This is something that actually appears to be a general pattern in the APIs. And is something that Ben has been using a bit

Reply:
From what I've read it performs the same conversion as the Number constructor but in my opinion is just a little cleaner and easier to read. I think the big difference is the unary plus can't parse bigints so if ids ever grow into the quadrillions it will need to be updated.